docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx

```diff
@@ -0,0 +1,79 @@
+---
+id: burn-hnt-to-maker-wallet
+hide_title: true
+sidebar_label: Burning HNT to Maker Wallet
+slug: /mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Burning HNT to Your Maker Wallet
+
+Any [approved Hotspot Maker](/mine-hnt/approved-makers) is responsible for supplying Data Credits (DCs) to the onboarding server for their Hotspots. They do this by burning HNT to their 
+specific maker address. 
+
+:::info 
+
+A full list of approved maker wallet addresses [can be seen via this API](https://onboarding.dewi.org/api/v2/makers).
+
+:::
+
+## Maker Wallet Address Integration
+
+As an approved Maker, you've been supplying the DeWi with the onboarding details for all the Hotspots you manufacture and distribute. In turn, the DeWi adds them to the onboarding server. When a Hotspot is onboarded by your end customer, the onboarding server knows to associate it with your Maker ID, and associated Maker Wallet address. If, when a Hotspot is onboarded, it's present in the onboarding server and you have ample Data Credits to pay the onboarding fees, the blockchain will add the Hotspot. And if you've [integrated your Hotspot in the Helium Mobile Wallet onboarding flow](/mine-hnt/hotspot-makers/hotspot-integration-testing), the Wallet knows to use your approved Maker Wallet address. At this point, the Data Credits will be flushed from your wallet address. 
```

Comment:
As an approved maker, you **will supply DeWi the onboarding** details...

Reply:
> And if you've integrated your Hotspot in the Helium Mobile Wallet onboarding flow

Change to "And if you've integrated your Hotspot in the Hotspot App onboarding flow

Reply:
> At this point, the Data Credits will be flushed from your wallet address. 
Change to "At this point, Data Credits will be consumed by the onboarding transactions (Add Hotspot and/or Assert Location) and deducted from your Maker wallet address.

Reply:
Fixed.